lxd/images.go

```diff
@@ -3566,7 +3566,7 @@ func autoSyncImagesTask(d *Daemon) (task.Func, task.Schedule) {
 		logger.Infof("Done synchronizing images across the cluster")
 	}
 
-	return f, task.Daily()
+	return f, task.Every(time.Hour)
```

Comment:
As we already have one of those `.Every(time.Hour)` can you add `Hourly()` as an alias in `task/schedule.go` and use it in both locations?

Reply:
Sure

Reply:
@stgraber https://github.com/lxc/lxd/pull/8646 as this is unrelated to the cluster fixes.